src/main/java/org/folio/circulation/resources/TenantActivationResource.java

```diff
@@ -21,8 +24,9 @@ public void register(Router router) {
   }
 
   public void enableModuleForTenant(RoutingContext routingContext) {
-    PubSubRegistrationService.registerModule(new WebContext(routingContext).getHeaders(),
-      routingContext.vertx())
+    Map<String, String> headers = new WebContext(routingContext).getHeaders();
+    PubSubRegistrationService.registerModule(headers,routingContext.vertx())
+      .thenCompose(b -> PubSubModuleRegistrationUtil.registerLogEventPublisher(headers, routingContext.vertx()))
```

Comment:
Why does the log publisher need to be a separate publisher to the module itself?